### ISSUE TITLE
Include path set in drushrc.php is ignored

### ISSUE THREAD

**Issue by mikeyp** [drush-ops/drush#1237]
I set a custom include path in drushrc.php, which I verified is being properly included. The options are even being merged into drush_get_context, but _drush_add_commandfiles() is never called with the search path specified in the drushrc.php.

**Comment by weitzman** [comment]
I wonder if this ever worked. I've only set --include on the CLI, not in
drushrc.

**Comment by mikeyp** [comment]
The example.drushrc.php has an example of this at the bottom: https://raw.githubusercontent.com/drush-ops/drush/master/examples/example.drushrc.php

The only problem is that this doesn't work when the drushrc.php is in sites/default: the drushrc.php *is* included, but not until command dispatch has started, which makes it too late to register an include path, and find the additional commands.

Setting the drushrc.php with --config works fine, since that includes it early enough in the bootstrap.

**Comment by weitzman** [comment]
Oh, there are a bunch of config items which can't be set when you put a drushrc in sites/default. This is by design.

**Comment by mikeyp** [comment]
So, is this documented anywhere?

**Comment by mikeyp** [comment]
It seem this broke in c7ea4b1d8ee29b467b3233b320e779535a9d0b8f as well.

**Comment by weitzman** [comment]
OK, it used to work. I don't think though that this was an intentional feature.
